src/Microsoft.PowerShell.Commands.Diagnostics/GetCounterCommand.cs

```diff
@@ -444,7 +443,7 @@ private void ProcessGetCounter()
                 _cultureAndSpecialCharacterMap.TryGetValue(culture.Name, out characterReplacementList);
             }
 
-            StringCollection allExpandedPaths = new StringCollection();
+            List<string> allExpandedPaths = new List<string>();
```

Comment:
Since it is small PR please move this to line 478 where it is used.

Reply:
We cannot move this declaration as it would then be out of scope on line 494.